odfuzz/generators.py

```diff
@@ -3,120 +3,227 @@
 import random
 import uuid
 import datetime
+import base64
+import time
 
 from odfuzz.constants import BASE_CHARSET, HEX_BINARY
 from odfuzz.encoders import EncoderMixin
 
-START_DATE = datetime.datetime(1900, 1, 1, 0, 0, 0)
+START_DATE = datetime.datetime(1970, 1, 1, 0, 0, 0)
 END_DATE = datetime.datetime(9999, 12, 31, 23, 59, 59)
 DATE_INTERVAL = (END_DATE - START_DATE).total_seconds()
 
 
 class EdmBinary:
+
+    def generate_body(value):
+        return base64.b64encode(value.encode()).decode()
```

Comment:
hardcoded base64 encode for body - why? IMHO not every service will expect that. This should be at least configurable, like the usage of the encoding in the URL 

Pls cover in the next PR.

Reply:
according to the JSON doc [here](https://www.odata.org/documentation/odata-version-2-0/json-format/) Binary is supposed to be base64 encoded for JSON representation. But if its strictly not the case for every service, then I agree it should be configurable.